features/bootstrap/FeatureContext.php

```diff
@@ -83,4 +83,16 @@ public function assertPageAddressAfterAllRedirection(string $url, int $ttl = 3,
             $this->assertPageAddressAfterAllRedirection($url, $ttl, ++$try);
         }
     }
+
+    /**
+     * @When I click the :cssElementSelector
+     */
+    public function clickElement($cssElementSelector)
```

Comment:
this new step is to force clicking on an element directly when it's not working with a basic check with the @javascript tag on

(thanks @ottaviano )